base/rhel9-python-3.9/Dockerfile

```
@@ -0,0 +1,40 @@
+FROM registry.redhat.io/rhel9/python-39:latest
+
+LABEL name="odh-notebook-base-rhel9-python-3.9" \
+      summary="Python 3.9 Red Hat Enterprise Linux 9 base image for ODH notebooks" \
+      description="Base Python 3.9 builder image based on Red Hat Enterprise Linux 9 for ODH notebooks" \
+      io.k9s.display-name="Python 3.9 RHEL9 base image for ODH notebooks" \
+      io.k9s.description="Base Python 3.9 builder image based on RHEL9 for ODH notebooks" \
```

```suggestion
      io.k8s.description="Base Python 3.9 builder image based on Red Hat Enterprise Linux 9 for ODH notebooks" \
```

Just to unify names we use across these descriptions.